test/e2e/data/kustomize/common-patches/cni/patch-cluster.yaml

```diff
@@ -1,12 +1,4 @@
 ---
-- op: add
-  path: /spec/bastion
-  value:
-    enabled: true
-    instance:
-      flavor: ${OPENSTACK_BASTION_MACHINE_FLAVOR}
-      image: ${OPENSTACK_BASTION_IMAGE_NAME}
-      sshKeyName: ${OPENSTACK_SSH_KEY_NAME}
```

Comment:
I think we've come full circle on this: we need to keep the Bastions for logging.